Services/README.md

```diff
@@ -0,0 +1,10 @@
+# Rules for the ./Services internal interface documentation
```

Comment:
Since this is THE README.md of the Services directory. I would like to see one short sentence in the beginning of the document shortly explaining, what the content of this directory is about.

Something like:
"Services provide general functionalities used in the modules (or in other services), e.g. the role based access system or the news system" (From the dev-Guide).

Further we might also add some differentiation from the sources dir. Something like.
"If parts can be almost completely abstracted from other ILIAS dependency, they SHOULD be placed in the src directory."

So we would have something like:
# Services
General description

## Guidelines for the ./Services internal interface documentation
....